### PR TITLE
236 docs rfc update the docs for the popsqlconsole integration

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

**Review by murrayju — APPROVED** [review]
🚀 lgtm

File: getting-started/services.md [hunk 1]
```diff
@@ -7,13 +7,17 @@ content_group: Getting started
...
 import Install from "versionContent/_partials/_cloud-installation.mdx";
-import CreateService from "versionContent/_partials/_cloud-create-service.mdx";
 import Connect from "versionContent/_partials/_cloud-connect.mdx";
-import CloudIntro from "versionContent/_partials/_cloud-intro.mdx";
+import CreateAHypertable from "versionContent/_partials/_create-hypertable.mdx";
 
 # Create your first Timescale service
 
-You use Timescale Console to create and manage Timescale services. Each service is a single 100% PostgreSQL database with usage-based storage.
+You use Timescale Console to create and manage `Ops` for your Timescale Cloud services, and securely manage
```
Comment (rahilsondhi) [inline getting-started/services.md:15]: This reads kinda funky to me, the `manage Ops` part.

What do you think of introducing it like:
- There are two modes in the Timescale Console, depending on what you're trying to accomplish
- Ops mode (short for operations) is where you can manage your services, add replicas, configure compression, change parameters, etc. 
- Data mode is where you can interact with your data, powered by PopSQL: write queries with autocomplete, visualize data with charts and dashboards, schedule queries and dashboards to create alerts or recurring reports, share queries and dashboards, and more.

The screenshots could also be side-by-side, and under each screenshot, you could describe each mode. That would help people visualize "ok Ops mode is for this kinda stuff, Data mode is for that kinda stuff". I used [v0.dev](https://v0.dev) to generate a sample mockup to convey what I mean:
image

File: getting-started/run-queries-from-console.md [hunk 2]
```diff
@@ -12,106 +12,70 @@ As Timescale Cloud is based on PostgreSQL, you can use lots of [different tools]
 connect to your service and interact with your data. To securely manage your date from inside
```
Comment (rahilsondhi) [inline getting-started/run-queries-from-console.md:12]: typo `manage your date`

File: getting-started/run-queries-from-console.md [hunk 3]
```diff
@@ -12,106 +12,70 @@ As Timescale Cloud is based on PostgreSQL, you can use lots of [different tools]
 connect to your service and interact with your data. To securely manage your date from inside 
 Console, Timescale offers the following Cloud SQL editors:
```
Comment (rahilsondhi) [inline getting-started/run-queries-from-console.md:13]: I know we call it `Cloud SQL editor` on the pricing page, but I don't know if it's useful on this docs page.

What if we just said it like:
- Within Timescale Console, there are two ways to run SQL queries against your database
- Within `Ops mode`, click on `SQL Editor`, and you can run ad-hoc ephemeral queries
  - This is useful for quick one-off tasks like creating an index on a small table or inspecting `pg_stat_statements`
- `Data mode` is a much richer experience powered by PopSQL where you can write queries with autocomplete, save them in folders, share them, create charts/dashboards, etc.

File: getting-started/run-queries-from-console.md [hunk 4]
```diff
@@ -176,25 +121,57 @@ a [read replica][readreplica] and send analytical queries there.
 
 If you'd like to prevent write operations such as insert or update. Instead 
 of using the `tsdbadmin` superuser, create a read-only user for your service and
-use that within PopSQL.
+use that in Data mode.
+
+
+
+## SQL editor
```
Comment (rahilsondhi) [inline getting-started/run-queries-from-console.md:128]: Maybe `Ops mode SQL editor` ?

File: getting-started/run-queries-from-console.md [hunk 5]
```diff
@@ -176,25 +121,57 @@ a [read replica][readreplica] and send analytical queries there.
...
+4. **Run a test query**
+
+   Type your query in the UI, then click `Run`. The results appear in the lower window.
+
+</Procedure>
+
 
 ## Cloud SQL editor licenses
 
 * **SQL editor**: free for anyone with a [Timescale Cloud account][create-cloud-account].
```
Comment (rahilsondhi) [inline getting-started/run-queries-from-console.md:159]: Maybe `Ops mode SQL editor` ?

File: _partials/_cloud-connect.md [hunk 6]
```diff
@@ -1,67 +1,56 @@
...
-
-    - **PopSQL**: the table is automatically displayed when you run the `CREATE` query.  
-    - **psql**: run the `\dt` command. You see the table listing for your service. To disconnect, type `exit`.
-
-
-Quick recap, you find configuration information about your
-services in [Services section in Timescale Console][tsc-portal], you find configuration and security information in your
-config file.
+Quick recap. You:
+- Manage your services in the [`Ops` in Timescale Console][portal-ops-mode]
```
Comment (rahilsondhi) [inline _partials/_cloud-connect.md:41]: Might be good to give some examples to drive the point home, like:
- Manage your services in `Ops mode` (eg add a replica, change parameters, etc)
- Analyze your data in `Data mode` (eg save queries, create charts/dashboards, etc)

File: use-timescale/services/service-overview.md [hunk 7]
```diff
@@ -11,16 +11,16 @@ cloud_ui:
 
 # About services
 
+You use Timescale Console to create and manage `Ops` for your Timescale Cloud services, and securely manage
```
Comment (rahilsondhi) [inline use-timescale/services/service-overview.md:14]: Similar comment, `manage Ops` reads funky

**Pull request merged**